### PR TITLE
Add a few hyperlinks to readme :)

### PR DESCRIPTION
**Pull request by bnzone**
### Changes in this Pull Request
- Added hyperlinks on first occurrence

### CODE REVIEW

**Comment by ascorbic** [comment]
PR #1! Thanks. I don't think we need to link every mention though. I think the first occurence is plenty

**Comment by bnzone** [comment]
> PR #1! Thanks. I don't think we need to link every mention though. I think the first occurence is plenty

Absolutely! First occurrence is enough 🙂

File: README.md [hunk 1]
```diff
@@ -43,6 +43,6 @@ I started with React because it's the one most obviously lacking a basic SSG, bu
 
 ## License
 
-Copyright © 2023 Matt Kane. Licenced under the MIT licence.
+Copyright © 2023 Matt Kane. Licensed under the MIT license.
```
Comment (ascorbic) [inline README.md]: I'm British

Comment (bnzone) [reply README.md]: My apologies! My [Spellchecker](https://github.com/streetsidesoftware/vscode-spell-checker) highlighted and I auto-corrected without a second thought 🙈

**Pull request merged**